binary/envconfig.json

```diff
@@ -0,0 +1,26 @@
+[{
+    "version": "1.0.0", 
+    "runtimeVersion" : "3.5-alpine", 
+    "kind": "environment", 
+    "status": "Stable", 
+    "name": "Fission Binary Environment",
```

Comment:
Should we add an attribute to effect if it has a builder or something like that, WDYT? `hasBuilder`?

Reply:
I am thinking how it will help. Should we have a boolean field or some field(s) to detail about the builder.